README.md

```diff
@@ -16,163 +16,25 @@
-- [Helm Workflow](https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service-helm.yml)
-- [Kompose Workflow](https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service-kompose.yml)
-- [Kustomize Workflow](https://github.com/actions/starter-workflows/blob/main/deployments/azure-kubernetes-service-kustomize.yml)
-
-### Show in Azure Portal
-
-Right click on your AKS cluster and click on **Show in Azure Portal** to easily navigate to AKS cluster overview page in Azure Portal.
-
-### Show Properties
-
-Right click on your AKS cluster and click on **Show Properties** to display the AKS cluster and agent pool properties like provisioning state, fqdn, k8s version, along with node properties like node version, vm type, vm size, o/s type, o/s disk size and nodes provisioning state.
-
-### Create cluster from Azure Portal
-
-Right click on your AKS sunscription and click on **Create cluster from Azure Portal** to easily navigate to AKS create cluster page in Azure Portal.
-
-### Start or Stop AKS cluster
-
-Right click on your AKS cluster and click on **Show Properties** to display the AKS cluster properties, within the page there will be **Stop/Start Cluster** button to perform the start or stop the cluster operation.
-
-![Start or Stop Cluster From Properties Webview](resources/aks-startstop-cluster.png)
-
-### Run Kubectl Commands from your AKS cluster
-
-Right click on your AKS cluster and click on **Run Kubectl Commands** to easily run few known kubectl commands on your cluster. Currently we have enable following kubectl commands for the AKS cluster:
-
-- Describe Services
-- Get All Pods
-- API Resources
-- Get Cluster Info
-- Get Node
-- Get All Events
-
-### Run Managed Cluster Operations from your AKS cluster
-
-Right click on your AKS cluster and click on **Managed Cluster Operations** to easily run few managed cluster operations on your cluster. Currently we have enable following operations:
-
-- Delete Cluster
-- Rotate Cluster Certificate
-
-### Run Kubernetes API Health Endpoints
-
-Right click on your AKS cluster and click on **Kubernetes API Health Endpoints** to easily run few [k8s cluster API endpoint health check](https://kubernetes.io/docs/reference/using-api/health-checks/). Currently we have enable following health check:
-
-- Healthz
-- Livez
-- Readyz
-
-### Deploy and Undeploy InspektorGadget
-
-Right click on your AKS cluster and click on **Inspektor Gadget** to easily deploy gadget into your cluster. User can easily one-click deploy and undeploy gadget from this feature.
-
-### Profile, Top and Snapshot Inspektor Gadget Commands
-
-After successfully deploying Inspektor Gadget, right-click on your AKS cluster and select **Inspektor Gadget** and choose **Gadget Commands** to easily use non-interactive Top, Profile or Snapshot commands for your cluster.
+## Documentation, Features and Development Information
```

Comment:
Do we need links for each feature here? I wonder if it might become onerous keeping them in sync? This also applies to the bullet list at the top.

Could we get away with just links to (some of) the top level doc sections (maybe Introduction, Installation/Quick start, Features)?